app/helpers/component-argument.js

```diff
@@ -9,7 +9,7 @@ import { helper } from '@ember/component/helper';
  */
 export function componentArgumentDisplay([argument]) {
   if (typeof argument === 'string') {
-    return JSON.stringify(argument);
+    return JSON.stringify(argument).replace(/^\"|\"$/g, '');
```

Comment:
At this point, this should probably just be `argument.replace(/\"/g, '\\"');`. Should probably leave a comment saying we are intentionally not wrapping it with quotes (doing it in the template), just escaping any interior quotes.

Reply:
I think this was a side effect of adding `JSON.stringify`. Before, the quotes were not part of the value.